packages/spor-react/src/input/NumericStepper.tsx

```diff
@@ -249,17 +252,18 @@ const AddIcon = (props: BoxProps) => (
   </Box>
 );
 
-const texts = createTexts({
-  decrementButtonAriaLabel: {
-    nb: "Trekk fra 1",
-    en: "Subtract 1",
-    nn: "Trekk frå 1",
-    sv: "Subtrahera 1",
-  },
-  incrementButtonAriaLabel: {
-    nb: "Legg til 1",
-    en: "Add 1",
-    nn: "Legg til 1",
-    sv: "Lägg till 1",
-  },
-});
+const texts = (stepSize: number) =>
+  createTexts({
+    decrementButtonAriaLabel: {
+      nb: `Trekk fra ${stepSize}`,
+      en: `Subtract ${stepSize}`,
+      nn: `Trekk frå ${stepSize}`,
+      sv: `Subtrahera ${stepSize}`,
+    },
+    incrementButtonAriaLabel: {
+      nb: `Legg til ${stepSize}`,
+      en: `Add ${stepSize}`,
+      nn: `Legg til ${stepSize}`,
+      sv: `Lägg till ${stepSize}`,
+    },
+  });
```

Comment:
Gjør det heller slik:
```suggestion
createTexts({
  decrementButtonAriaLabel(stepSize) {
    nb: `Trekk fra ${stepSize}`,
    en: `Subtract ${stepSize}`,
    nn: `Trekk frå ${stepSize}`,
    sv: `Subtrahera ${stepSize}`,
  },
  incrementButtonAriaLabel(stepsize) {
    nb: `Legg til ${stepSize}`,
    en: `Add ${stepSize}`,
    nn: `Legg til ${stepSize}`,
    sv: `Lägg till ${stepSize}`,
  }
})
```

Det er slik de er tiltenkt brukt i alle fall. Så blir bruken slik: `t(texts.incrementButtonAriaLabel(stepSize))`